### PR TITLE
fix italian translations

### PR DESCRIPTION
**Pull request by camilavcoutinho**
#### What is the purpose of this pull request?

Fix Italian translations

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by Silvia-Spela — APPROVED** [review]

**Review by Silvia-Spela — APPROVED** [review]

**Comment by lucasecdb** [comment]
@camilavcoutinho can you rebase? there is a conflict in the changelog

File: messages/it.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
   "address-form.field.country": "Stato",
   "address-form.field.postalCode": "Codice postale",
   "address-form.field.street": "Via",
-  "address-form.field.addressLine1": "Indirizzo",
+  "address-form.field.addressLine1": "Via e numero civico",
```
Comment (lucasecdb) [inline messages/it.json:42]: @camilavcoutinho I think you lost some of the modifications :-/

this should be only "Via" right? and for the number below it should be "Numero civico"

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: messages/it.json (hunk 1 above)
Comment (camilavcoutinho) [reply messages/it.json:42]: Yeah! I think when I did the rebase, I lost the modifications. Sorry about that! I'll fix it right away.

**Review by lucasecdb — APPROVED** [review]
nice! ✨

**Review by Silvia-Spela — APPROVED** [review]

**Pull request merged**